scripts/schemas/twister/testsuite-schema.yaml

```diff
@@ -145,6 +145,11 @@ mapping:
         matching: "all"
         sequence:
           - type: str
+      "required_snippets":
```

Comment:
might it be just `snippets` ? or are there other types of snippets possible e.g. 'optional'. If so,  then shouldn't it be `snippets_required` to follow the naming convention here ?

Reply:
The "required" part is to make it clear that the test will only run with them. I don't envisage having support for optional snippets, they will either be required and applied or not available and test will not run